_services_projects/consent_based_sharing.md

```diff
@@ -0,0 +1,19 @@
+---
+agency: Department of Internal Affairs
+title: Consent Based Data Sharing
+subtitle: Prototyping ways for people to share their own government data with other agencies and organisations.
+permalink: /projects/consent-based-sharing/
+redirect_from: /projects/consent-based-sharing/
+excerpt: "Giving people visibility and control over their government data."
+image: /assets/img/projects/consent-based-data-sharing.png
```

Comment:
the dark blue overlayed images are hard for people without perfect eyesight.

> Could we move to full colour? Would it work with the theme?

Reply:
They aren't very illustrative are they - can we add this as an issue and come back to it? I wouldn't want to have just this one different from the others